### PR TITLE
collectd: enable AllPortsSummary for tcpconns plugin

### PR DESCRIPTION
**Pull request by mstorchak**
Maintainer: @jow- , @hnyman 
Compile tested: aarch64_cortex-a53, ipq807x/generic, r22604+4-a247f49794
Run tested: aarch64_cortex-a53, ipq807x/generic, r22604+4-a247f49794

Description:
enable summarized stats for tcp connections, https://collectd.org/wiki/index.php/Plugin:TCPConns#Summary_of_all_ports

### CODE REVIEW

**Comment by mstorchak** [comment]
Screenshot 2023-05-29 at 23-09-59 Net - Grafana

File: utils/collectd/files/collectd.uci [hunk 1]
```diff
@@ -188,7 +188,8 @@ config globals 'globals'
 
 #config plugin 'tcpconns'
 #	option enable '0'
-#	list ListeningPort '0'
+#	option 'AllPortsSummary' '0'
+#	option ListeninngPort '0'
```
Comment (hnyman) [inline utils/collectd/files/collectd.uci:192]: Likely a typo:  "ListeninngPort" 
(it was right before changes...)

Comment (mstorchak) [reply utils/collectd/files/collectd.uci:192]: it's boolean in json. Should it really be "list" here?

Comment (mstorchak) [reply utils/collectd/files/collectd.uci:192]: https://collectd.org/wiki/index.php/Plugin:TCPConns#Single_port

Comment (mstorchak) [reply utils/collectd/files/collectd.uci]: Ah, I see the typo. Thanks!

Comment (hnyman) [reply utils/collectd/files/collectd.uci]: Based upstream source code, it is a boolean flag. Should be "option", I think.

And I now notice that also the original spelling was wrong (missing s):
not ListeningPort , 
not ListeninngPort, 
but ListeningPorts

https://github.com/collectd/collectd/blob/collectd-5.12/src/tcpconns.c#L260-L261

Comment (mstorchak) [reply utils/collectd/files/collectd.uci]: yeah, it's ListeningPort**s** indeed.

**Comment by hnyman** [comment]
looks ok to me. 
I will merge it.

**Pull request merged**